clients/ops/admin-ui/src/features/common/DaysLeftTag.tsx

```diff
@@ -0,0 +1,37 @@
+import { Tag } from "@fidesui/react";
+import React from "react";
+
+type DaysLeftTagProps = {
+  days_left: number | undefined;
+  include_text: boolean;
+};
+
+const DaysLeftTag = ({ days_left, include_text }: DaysLeftTagProps) => {
+  if (!days_left) {
+    return null;
+  }
+
+  let backgroundColor = "";
+
+  if (days_left >= 10) {
```

Comment:
I would suggest using a switch case range statement here instead of the multiple if statements:

https://tutorial.eyehunts.com/js/javascript-switch-case-range-example-code/